### PR TITLE
add strapi v4 files

### PR DESCRIPTION
**Pull request by hacktivist123**
In this pull request, I added the required files for strapi v4 templates.

### CODE REVIEW

**Comment by hacktivist123** [comment]
PR on the standalone Repo: https://github.com/platformsh-templates/strapi4/pull/1

File: templates/strapi4/.platform.template.yaml [hunk 1]
```diff
@@ -0,0 +1,29 @@
...
+
+    notes:
+        - heading: "Features"
+          content: |
+              Node.js 12<br />
+              PostgreSQL 12<br />
+              Automatic TLS certificates<br />
+              yarn-based build<br />
+initialize:
+    repository: git://https://github.com/platformsh-templates/strapi-v4-postgresql.git@master
```
Comment (chadwcarlson) [inline templates/strapi4/.platform.template.yaml:26]:
```suggestion
    repository: git://https://github.com/platformsh-templates/strapi4.git@master
```

File: templates/strapi4/files/.platform.app.yaml [hunk 2]
```diff
@@ -0,0 +1,70 @@
...
+# Build dependencies.
+dependencies:
+  nodejs:
+    yarn: "1.22.5"
+
+# The hooks executed at various points in the lifecycle of the application.
+hooks:
+  build: |
+    # Download dependencies and build Strapi.
+    yarn
```
Comment (chadwcarlson) [inline templates/strapi4/files/.platform.app.yaml:25]:
```suggestion
    yarn --frozen-lockfile
```

File: templates/strapi4/files/.platform.app.yaml [hunk 3]
```diff
@@ -0,0 +1,70 @@
...
+    source: local
+    source_path: cache
+
+  # Mount .tmp file in the app folder for strapi
+  ".tmp":
+    source: local
+    source_path: app
+
+  # Mount database folder for strapi
+  "database":
```
Comment (chadwcarlson) [inline templates/strapi4/files/.platform.app.yaml:58]: This is only necessary for SQLite, yes?

File: templates/strapi4/files/README.md [hunk 4]
```diff
@@ -0,0 +1,13 @@
+# Strapi v4 template for Platform.sh
+
```
Comment (chadwcarlson) [inline templates/strapi4/files/README.md:2]: Missing Deploy on Platform.sh button

File: templates/strapi4/files/.platform.app.yaml (hunk 3 above)
Comment (hacktivist123) [reply templates/strapi4/files/.platform.app.yaml:58]: I'm not sure yet

File: templates/strapi4/files/README.md (hunk 4 above)
Comment (hacktivist123) [reply templates/strapi4/files/README.md:2]: How do i generate that by the way?

Comment (hacktivist123) [reply templates/strapi4/files/README.md:2]: okay, will do so now

Comment (hacktivist123) [reply templates/strapi4/files/README.md:2]: done

File: templates/strapi4/files/README.md [hunk 5]
```diff
@@ -0,0 +1,54 @@
...
+<p align="center">
+<a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/platformsh/template-builder/master/templates/strapi4/.platform.template.yaml&utm_content=strapi&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
```
Comment (chadwcarlson) [inline templates/strapi4/files/README.md:4]:
```suggestion
<a href="https://console.platform.sh/projects/create-project?template=https://raw.githubusercontent.com/platformsh/template-builder/master/templates/strapi4/.platform.template.yaml&utm_content=strapi4&utm_source=github&utm_medium=button&utm_campaign=deploy_on_platform">
```

File: templates/strapi4/files/.platform.app.yaml (hunk 2 above)
Comment (chadwcarlson) [reply templates/strapi4/files/.platform.app.yaml:25]: Shouldn't we change this? Doesn't _not_ using this flag modify the lock file on install?

Comment (hacktivist123) [reply templates/strapi4/files/.platform.app.yaml:25]: yes, i did exactly that yesterday, you removed the frozen lockfile tag

Comment (chadwcarlson) [reply templates/strapi4/files/.platform.app.yaml:25]: I did?

**Comment by chadwcarlson** [comment]
https://github.com/platformsh/template-builder/issues/475

**Pull request merged**